### ISSUE TITLE
result

### ISSUE THREAD

**Issue by wyfsmn** [RazvanDu/DUCK-Net#10]
In the CVC Colondb dataset, the obtained dice_ Test: 0.8902772313306522, why does it differ so much from the results in the paper

**Comment by RazvanDu** [comment]
Did you use the weights we provided for the model?

**Comment by wyfsmn** [comment]
Is it not possible to achieve the effect of the paper without weights, and will it be much worse?

**Comment by RazvanDu** [comment]
It is certainly possible but if you expect to get the same results as we did you have to use our settings and our data set split.

**Comment by wyfsmn** [comment]
cvc-colondb
dice_train: 0.9314982832549971 dice_valid: 0.9496475459978354 dice_test: 0.8709802621263646
What is the reason why the code achieved good results on the validation set, but did not achieve the expected results on the test set? How can I solve this problem?

**Comment by maenstru56** [comment]
Hello,

The most likely reason for the different values you obtained is that your setup is different, please make sure to:
- Install the proper versions of the required libraries ([requirements.txt](https://github.com/RazvanDu/DUCK-Net/blob/main/requirements.txt)). It's important to use the same versions we used because the backend implementation of certain functions may differ between versions and this may impact the results.
- Use the [data split](https://github.com/RazvanDu/DUCK-Net#data-sets) and [model weights](https://github.com/RazvanDu/DUCK-Net#model-weights) we provided. 
- Make sure to use the same testing methodology. The code can be found in the last cell of the Jupyter Notebook: [ModelNotebook.ipynb](https://github.com/RazvanDu/DUCK-Net/blob/main/ModelNotebook.ipynb).

If you do the steps above, the results you obtain should be the same as the ones we reported in the paper. 

In case you want to use a different setup to test our model, feel free to use the resulting values you obtain in that context, but make sure to mention the said context.

**Comment by wyfsmn** [comment]
I noticed that you provided two types of datasets, one is a dataset split into three parts, and the other is a dataset with only one part. Where should I use the dataset for training? The code seems to split the entire dataset instead of using the three part dataset you provided

**Comment by maenstru56** [comment]
Yes, the code we provided is more general, so that it can be used for different types of datasets and can be easily adapted. 

If you want to replicate our results, please use the split datasets ([here](https://drive.google.com/drive/folders/1kg9XImzrd9PpTtleQSz6l8uq82LV1sjV)) and adapt the code to separately load the training, validation and testing data. By doing this, you don't need the "splitting" anymore, you can use the "ImageLoader2D.load_data" function to directly load all 3 data sub-sets (training/validation/testing).
